### ISSUE TITLE
python-language-python.exe version 3.25.0 for windows quarantined by antivirus software

### ISSUE THREAD

**Issue by nicro950** [pulumi/pulumi#9064]
## Hello!

- Vote on this issue by adding a 👍 reaction
- To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already)

## Issue details

On Friday, we started getting reports from antivirus software that it had quarantined python-language-python.exe version ~~2.5.0~~ 3.25.0 for windows, resulting in builds started to fail on build agents. Is there any known problem with this?

EDIT: 
Updated to correct version number

**Comment by nicro950** [comment]
@Frassle  Ahh, sorry, not quite sure why I wrote that version number. It is for version 3.25.0 not 2.5.0, probably misspelled some of the dots, and copied a lot of text. I will updated the header as well.

**Comment by nicro950** [comment]
Our current solution was to downgrade to pulumi version 3.24.1, since that has not been quarantined, but the message from the Antiviurs software is: 

Program:Win32/Cayunamar.A!ml
Alert level: High
Status: Quarantined
Date: 25.02.2022
Category: Potentially Unwanted software
Details: This program has potentially unwanted behavior

**Comment by Frassle** [comment]
I downloaded the release from https://github.com/pulumi/pulumi/releases/download/v3.25.0/pulumi-v3.25.0-windows-x64.zip and ran the extracted files through windows defender anti-virus and it didn't find anything. 

Can you try downloading from the github link above.
Let us know which anti-virus software your using that's flagging this.

**Comment by nicro950** [comment]
It was windows defender that triggered on the file in the first place, but it looks like it is no longer triggering, so i guess it was a mistake in the detection and has been fixed since. 
I also downloaded the file from that URL, and that was the same file, with the same checksum, that I had downloaded earlier. 

But then the answer to my original question is probably that there is no more information regarding this incident. Thanks for your time anyway, this issue can probably be closed now
